scripts/analysis.py

```diff
@@ -586,12 +736,12 @@ def stacked_bar_chart(dictionary, timestep,
         else:
             plot = plt.bar(left=init_year + (timestep/12),
                            height=dictionary[key],
-                           width=0.5,
+                           width=0.1,
                            color=cm.viridis(1.*color_index/len(dictionary)),
                            edgecolor='none',
                            bottom=prev,
```

Comment:
during the first go-round, it seems like prev will be completely undefined. (you define it as an empty string in a previous function, but that definition is never used and goes out of scope before this usage). Surely it should be an array of zeros instead? (see np.zeros)